src/openklant/components/klantinteracties/api/tests/test_filters.py

```diff
@@ -630,6 +630,33 @@ def test_filter_was_partij_nummer(self):
 
             self.assertEqual(response.json()["count"], 0)
 
+    def test_digitale_adressen_inclusion_param(self):
+        betrokkene = BetrokkeneFactory.create()
+        response = self.client.get(self.url, data={"expand": "digitaleAdressen"})
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        results = response.json()["results"]
+        self.assertEqual(len(results), 1)
+        self.assertEqual(results[0]["_expand"], {"digitaleAdressen": []})
+
+        digitaal_adres = DigitaalAdresFactory(
+            betrokkene=betrokkene,
+            adres="test",
+            soort_digitaal_adres="email",
+        )
+
+        response = self.client.get(self.url, data={"expand": "digitaleAdressen"})
```

Comment:
can you also add a test to verify that this works on the detail endpoint for `betrokkene`?